R/datatypes-s3-base.R

```diff
@@ -38,7 +38,12 @@
 
 #' @export
 print.polars_dtype <- function(x, ...) {
-  x$`_dt`$as_str(abbreviated = FALSE) |>
+  format(x, abbreviated = FALSE) |>
     writeLines()
   invisible(x)
 }
+
+#' @export
+format.polars_dtype <- function(x, abbreviated = FALSE, ...) {
```

Comment:
- Doesn't the additional argument need to be after `...`?
- Can you add documentation to help us understand how to use it?

Reply:
Done, but since the other methods do not produce an Rd file, I don't think this one should too.

I've also added a check for empty dots.